js/apps/system/_admin/aardvark/APP/react/src/components/pure-css/form/Tag.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+
+type TagProps = {
+  label?: string;
+  template?: string;
+};
+
+const Tag = ({ label, template }: TagProps) => {
+
+    if (template === 'transparent') {
+        return <>
+            {label ?
+                <div
+                    style={{
+                        'background': 'transparent',
+                        'color': '#333333',
+                        'padding': '4px 0px',
+                        'borderRadius': '4px',
+                        'width': 'fit-content',
+                        'margin': '4px'
+                        }}
+                >
+                    {label}
+                </div> :
+                null
+            }
+        </>;
+    }
+
+    return <>
+        {label ?
+            <div
+                style={{
+                    'background': '#404A53',
+                    'color': '#ffffff',
+                    'padding': '4px 8px',
+                    'borderRadius': '4px',
+                    'width': 'fit-content',
+                    'margin': '4px'
+                    }}
+            >
+                {label}
+            </div> :
+            null
+        }
+    </>;
+};
+
+export default Tag;
```

Comment:
switch to using Tag from Chakra

Reply:
Resolved within https://github.com/arangodb/arangodb/pull/18199/commits/d4defa51db6856c5cc7d3b075c4a613051d5180b